libs/scully/src/lib/utils/captureMessage.ts

```diff
@@ -1,30 +1,35 @@
-import * as sentry from '@sentry/node';
-import { readDotProperty } from './scullydot';
+// import * as sentry from '@sentry/node';
```

Comment:
can we remove this?

Reply:
not yet, we should add an toto here too.